include/OpenColorIO/OpenColorTransforms.h

```diff
@@ -867,6 +887,81 @@ class OCIOEXPORT LogAffineTransform : public Transform
 extern OCIOEXPORT std::ostream & operator<<(std::ostream &, const LogAffineTransform &);
 
 
+//!rst:: //////////////////////////////////////////////////////////////////
+
+//!cpp:class::  Same as :cpp:class:`LogAffineTransform` but with the addition of a linear segment
+// near black. This formula is used for many camera logs (e.g., LogC) as well as ACEScct.
+//
+// * The linSideBreak specifies the point on the linear axis where the log and linear
+//   segments meet.  It must be set (there is no default).  
+// * The linearSlope specifies the slope of the linear segment of the forward (linToLog)
+//   transform.  By default it is set equal to the slope of the log curve at the break point.
+//
+class OCIOEXPORT LogCameraTransform : public Transform
+{
+public:
+    //!cpp:function::
+    static LogCameraTransformRcPtr Create();
+
+    //!cpp:function::
+    virtual const FormatMetadata & getFormatMetadata() const noexcept = 0;
+    //!cpp:function::
+    virtual FormatMetadata & getFormatMetadata() noexcept = 0;
+
+    //!cpp:function:: Checks if this exactly equals other.
+    virtual bool equals(const LogCameraTransform & other) const noexcept = 0;
+
+    //!cpp:function::
+    virtual double getBase() const noexcept = 0;
+    //!cpp:function::
+    virtual void setBase(double base) noexcept = 0;
+
+    //!rst:: **Get/Set values for the R, G, B components**
+    //
+
+    //!cpp:function::
+    virtual void getLogSideSlopeValue(double(&values)[3]) const noexcept = 0;
```

Comment:
Just a general comment, not specific to this PR. I'm noticing we have some inconsistency in Transform method naming in the API. Some set/set methods use the `Value` suffix, and some don't. My opinion is that Value is unnecessary in these method names. So:

`getLogSideSlopeValue` could be `getLogSideSlope`

Again, not suggesting this be changed now necessarily, but would be good to do a consistency pass on the API before v2 releases.